pythonCode/create_MP2RAGE_json.py

```diff
@@ -0,0 +1,136 @@
+"""
+WARNING:
+This script was created in October 2018 when the specification
+for MP2RAGE files was not fully finalized. Please double check with
+latest version of the specs to make sure this is accurate.
+
+This script will create the JSON files required for an MP2RAGE file.
+To function it requires a BIDS data set with properly named image files:
+The script will through all the folders of a BIDS data set and will create
+the JSON files in every folder where it finds a file ending
+with '_MPRAGE.nii.gz'.
+
+Created by RG 2018-10-03
+"""
+
+import os
+import json
+from shutil import copyfile
+from collections import OrderedDict
+
+'''
+DEFINE CONTENT OF JSON FILES
+'''
+# sub-*_inv-1_MPRAGE.json
+data_inv_1 = OrderedDict()
+data_inv_1['InversionTime'] = '900'  # ms
+data_inv_1['FlipAngle'] = '5'
+
+# sub-*_inv-2_MPRAGE.json
+data_inv_2 = OrderedDict()
+data_inv_2['InversionTime'] = '2750'  # ms
+data_inv_2['FlipAngle'] = '3'
+
+# sub-*_T1map.json
+# sub-*_T1w.json
+data_T1 = OrderedDict()
+data_T1['EstimationMethod'] = 'Marques et al., 2013'
+
+
+# sub-*_MPRAGE.json
+data_MP2RAGE = OrderedDict()
```

Comment:
Same stylistic comment is above -- you can really see the difference here with many fields !